src/love.cpp

```diff
@@ -141,6 +141,17 @@ enum DoneAction
 	DONE_RESTART,
 };
 
+static void print_usage()
+{
+	printf("usage:\n"
+		"    love --version                  prints LÖVE version and quits\n"
+		"    love --help                     prints this message and quits\n"
+		"    love path/to/gamedir            runs the game from the given directory which contains main.lua file\n"
+		"    love path/to/packagedgame.love  runs the packaged game from the provided .love file\n"
+		"    love path/to/main.lua           runs the game from the given .lua file\n"
```

Comment:
Maybe this example could use a different filename than `main.lua`, so it's more clear that the name doesn't need to be that:
```suggestion
		"    love path/to/file.lua           runs the game from the given .lua file\n"
```